advisories/a50587.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Technical Advisory 50587
+summary: TRUNCATE prevents table renaming
+toc: true
+---
+
+Publication date: June 29, 2020.
+
+## Description
+
+When executing a [`TRUNCATE`](https://www.cockroachlabs.com/docs/stable/truncate.html) statement on table, CockroachDB creates a new table, with a new descriptor ID, copies the contents from the old table to the new table, and then [drops](https://www.cockroachlabs.com/docs/stable/drop-table.html) the old table. When the old table is dropped, a [drop job](https://www.cockroachlabs.com/docs/stable/online-schema-changes.html) is created to asynchronously clean up the table's data. In v19.2, v19.1, and prior versions, a job ID is assigned to the old table prior to copying its contents to the new table. This means that the new table descriptor will have a populated drop job ID. This drop job ID prevents the table from being [renamed](https://www.cockroachlabs.com/docs/v20.1/rename-table.html).
+
+## Statement
+
+This bug has been fixed in versions v19.1.11, v19.2.9, and v20.1.0. All users are encouraged to upgrade (see [Mitigation](#mitigation) below).
```

Comment:
I am not sure the release versions of the said maintenance releases. v20.1 is not impacted by this bug.